lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -578,21 +600,29 @@ def _get_user_count(self, query_args, cache_time=3600):
         hasher = hashlib.md5()
         for arg in query_args:
             hasher.update(bytes(arg))
+        if annotations:
```

Comment:
I think we can just rip out all of this hashing logic, as it seems like the LMS already hashes its cache keys when they're over a certain length via [this](https://github.com/edx/edx-platform/blob/master/common/djangoapps/util/memcache.py#L32) mechanism, per [this](https://github.com/edx/edx-platform/pull/21231/files#r308348836) comment from Cale. I was going to rip it out separately, but maybe now would be a good chance since you're modifying the hashing code here.